### ISSUE TITLE
Add option for inclusive end date times

### ISSUE THREAD

**Issue by knackjason** [chronotruck/vue-ctk-date-time-picker#88]
Can we get an option for inclusive end date times in ranges? The shortcuts seem to make the end date time inclusive (ex. `2018-10-15T23:59:59-05:00`) but this isn't the case when you select a range manually. What I'm getting back looks like

```json
{
  "start": "2018-12-02T00:00:00-05:00",
  "end": "2018-12-04T00:00:00-05:00"
}
```

**Comment by knackjason** [comment]
I also just noticed that picking a shortcut doesn't make the start date time "inclusive" (aka it uses the current time instead of setting the time to `00:00:00`). Since we're only selecting dates in range mode, it would make sense to for the start time to be `00:00:00` and the end time to be `23:59:59`, regardless of the range being selected via a shortcut or manually. Thoughts?

**Comment by LouisMazel** [comment]
Hi, 

To always have the same output format, use the `format` props. 
Ex : `format="YYYY-MM-DD"`

Time is not managed in range mode

Thanks

**Comment by knackjason** [comment]
So maybe this wasn't clear in my earlier comments but I'm not referring to the output format of the date (it's fine how it is). Rather, I'm asking about the inconsistent behavior of the time portion of the output when selecting a range. It's true that we don't select the time in range mode but time is still included in the output.

So there are two parts to my comments. I probably should have split them into two different tickets. Sorry about that.

1. Can we get an option (or just make it the default behavior) to always make the end date of a range inclusive? If I use one of the shortcut buttons, the end date is inclusive (ex.``{ "start": "2018-12-02T00:00:00-05:00", "end": "2018-12-04T23:59:59-05:00" }`) but this isn't the case when I manually select a range (ex. if I select Dec 2 - Dec 4, I get back `{ "start": "2018-12-02T00:00:00-05:00", "end": "2018-12-04T00:00:00-05:00" }`, meaning the 4th is not actually included in my range).

2. ~Using a shortcut button doesn't start the range at `00:00:00` on the first day of the range. Instead it seems to tack the current time onto the first day of the range.~ I just checked this again and am not able to reproduce it. Maybe it was just a hiccup. :man_shrugging: 

So this really isn't a question regarding formatting but rather a request to make the end dates of the range consistent (and preferrably inclusive), whether the range is selected manually or with a shortcut button.

**Comment by knackjason** [comment]
Any chance this ticket can be reopened given the above explanation? If not, I'll just continue to work around it with my hacky solution of "fixing" the time portion of the end date time after selecting a range.

**Comment by LouisMazel** [comment]
Hi @knackjason,

I fixed this in '2.0.0-rc8' version (beta)
Release : https://github.com/chronotruck/vue-ctk-date-time-picker/releases/tag/2.0.0-rc8

![range date formatting](https://pouch.jumpshare.com/preview/nNOPN9RkMnicfvRF7lj3DKaFc2Qy3AVHIWR39nXQ-irzseGUpogTPg9A_TX-vRDnEKfn0gDKiD9lvXQOupi_WVrWRC8tYABFuekS8mwDWCY)

**Comment by knackjason** [comment]
Cool, thank you! I'll take a look at it when I get a chance.

Thanks again for your prompt responses!
